modules/RTC/RTC.js

```diff
@@ -327,14 +336,17 @@ export default class RTC extends Listenable {
 
     /**
      * Sets the maximum video size the local participant should receive from
-     * remote participants. Will no-op if no data channel has been established.
+     * remote participants. Will cache the value and send it through the channel
+     * once it is created.
      *
      * @param {number} maxFrameHeightPixels the maximum frame height, in pixels,
      * this receiver is willing to receive.
      * @returns {void}
      */
     setReceiverVideoConstraint(maxFrameHeight) {
-        if (this._channel) {
+        this._maxFrameHeight = maxFrameHeight;
+
+        if (this._channel && this._channelOpen) {
```

Comment:
shouldn't there be some logic somewhere that waits on the channel open event and, when it fires, sends the message we tried to send it before the channel opened?

Reply:
That's probably incredibly crucial.